ydb/docs/en/core/maintenance/manual/backup-collections.md

```diff
@@ -0,0 +1,311 @@
+- Listing backups chronologically using `ydb scheme ls .backups/collections/<collection_name>/ | sort`
+- Identifying backup types by directory name suffix (`_full` or `_incremental`)
+- Ensuring you delete complete chains (all backups from a full backup through its incrementals)
+
+{% endnote %}
+
+### Manual cleanup strategies {#manual-cleanup-strategies}
+
+**Safe cleanup approach:**
+
+1. Create new full backup.
+2. Verify new backup is complete.
+3. Delete old backup chains (full backup + all its incrementals).
+4. Never delete partial chains.
+
+**Example cleanup workflow:**
+
+```bash
+# 1. Create new full backup
+ydb yql -s "BACKUP \`shop_backups\`;"
+
+# 2. Wait for completion and verify
+ydb operation list incbackup
+
+# 3. Browse backup structure
+ydb scheme ls .backups/collections/shop_backups/ | sort
+
+# 4. Remove old backup directories (entire chains only)
+ydb scheme rmdir -r .backups/collections/shop_backups/20250208141425Z_full/
+ydb scheme rmdir -r .backups/collections/shop_backups/20250209141519Z_incremental/
+ydb scheme rmdir -r .backups/collections/shop_backups/20250210141612Z_incremental/
+# ... (remove all related incrementals in the chain)
+```
+
+### Retention policies {#retention-policies}
+
+Implement retention policies based on:
+
+- **Business requirements**: How long data must be retained.
+- **Storage costs**: Balance retention with storage usage.
+- **Recovery needs**: Typical recovery scenarios and timeframes.
+- **Compliance**: Legal or regulatory requirements.
+
+**Example retention policy:**
+
+- Keep daily backups for 30 days.
+- Keep weekly backups for 12 weeks.
+- Keep monthly backups for 12 months.
+- Keep yearly backups for 7 years.
+
+## Backup validation and verification {#validation}
+
+### Post-backup validation {#post-backup-validation}
+
+After backup completion:
+
+1. **Verify operation success**: Check operation status.
+2. **Validate backup structure**: Browse backup directories.
+
+### Backup testing procedures {#backup-testing-procedures}
+
+Regularly test backup restoration:
+
+1. **Test environment**: Use separate test environment.
+2. **Sample restoration**: Restore subset of data.
+3. **Full restoration test**: Periodically test complete restoration.
+4. **Performance testing**: Measure restoration time and resources.
+
+## Recovery and restoration {#recovery}
+
+### Export and import restoration {#export-import-restoration}
+
+For complete disaster recovery, use export/import operations:
+
+```bash
+# Export backup collection from source
+ydb tools dump -p .backups/collections/shop_backups -o shop_backups_export
+
+# create backup collection in target database
+# CREATE BACKUP COLLECTION `collection_name`
+#     ( TABLE `table_path` [, TABLE `table_path` ...] )
+# WITH ( STORAGE = 'cluster', INCREMENTAL_BACKUP_ENABLED = 'true' );
+
+# Import to target database
+ydb tools restore -i shop_backups_export -d /Root/restored_db
```

Comment:
What to do if you only have one database? How to restore to a different folder or different filenames?